README.rst

```diff
@@ -71,9 +71,9 @@ In the example pillar above:
 ``docker.compose``
 ------------------
 
-Add support for using `Docker Compose <https://docs.docker.com/compose/>`_
+Installs `Docker Compose <https://docs.docker.com/compose/>`_
 (previously ``fig``) to define groups of containers and their relationships
-with one another.
+with one another. Use `docker.compose-ng` to run `docker-compose`.
```

Comment:
Did you want to add this because it was not obvious that `docker.compose-ng` is for running Docker Compose?